### ISSUE TITLE
camera_parameters and image_preprocessing

### ISSUE THREAD

**Issue by g-guenther** [NeXus-for-HELPMI/POLARIS_ExpData_Example#3]
This is rather a question: Do the NXcollection groups 'camera_parameters' and 'image_preprocessing' relate to the 'image' field of the instrument/detector group?

If so, I would recommend to shift these NXcollection groups to the instrument/detector group. Moreover, check if any of these terms can be translated/mapped to predefined NeXus terms, e.g.

- camera_parameters/exposuretimeabs to NeXus terms '/entry/instrument/detector/real_time' or '/entry/instrument/detector/count_time', respectively.
- camera_parameters/gain or camera_parameters/gain_auto to the NeXus term '/entry/instrument/detector/gain_setting', if applicable.

**Comment by keale** [comment]
Yes, the NXcollection groups "camera_parameters" and "image_preprocessing" refer to the "image". It is a good idea to move these to the detector.  

- Q for preprocessing: how I can link to a background image? Where BG image should be stored? I don't like to store it with each measurement.
- Regarding exptime <-> real_time: it only makes sense if I would specify a mapping rule. For me it is not intuitive to link real_time with exposuretime.
- One way or another, we need an extension for measuring devices. Even for cameras, there are a lot of parameters that certainly cannot be mapped to NeXus predefined terms

UPDATE: 
if I understand NXdetector manualcorrectly, most close parameter to exposuretime is the [count-time](https://manual.nexusformat.org/classes/base_classes/NXdetector.html#nxdetector-count-time-field).

**Comment by g-guenther** [comment]
> how I can link to a background image? Where BG image should be stored? I don't like to store it with each measurement.

In NXdetector it is intended to store the background image with your sample measurements but setting 'image_key'=4. If you want to store the background image separately, you could add another NXdetector with separate data field and separate 'image_key'=4 for the background image (or a link to NXdetector of another NXentry containing the background measurement). Would that fit?
